packages/serverpod_flutter/test/localhost_test.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:serverpod_flutter/src/localhost.dart';
+
+void main() {
+  test('returns 10.0.2.2 on Android apps', () {
```

Comment:
Suggestion: Format these tests following the "Given when then" pattern.

```suggestion
  test('Given target platform is android when reading localhost then 10.0.2.2 is returned', () {
```